frontend/packages/kubevirt-plugin/src/topology/TopologyVmDetailsPanel.tsx

```diff
@@ -28,9 +28,8 @@ type LoadedTopologyVmDetailsPanelProps = TopologyVmDetailsPanelProps & {
 const LoadedTopologyVmDetailsPanel: React.FC<LoadedTopologyVmDetailsPanelProps> = observer(
   ({ loaded, vmNode, templates }) => {
     const vmData = vmNode.getData();
-    const { pod } = vmData.data.vmStatusBundle;
     const vmObj = vmData.resource as VMKind;
-    const { vmi, vmStatusBundle } = vmData.data;
+    const { vmi, vmStatusBundle, pods } = vmData.data;
```

Comment:
can you add these fixes to `TopologyVmResourcesPanel.tsx` aswell?

Reply:
sure :)